x/conflict/types/relay_finalization_metadata.go

```diff
@@ -1,38 +1,72 @@
 package types
 
 import (
+	"encoding/json"
+	fmt "fmt"
+
 	tendermintcrypto "github.com/cometbft/cometbft/crypto"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/utils"
 	"github.com/lavanet/lava/utils/sigs"
 	pairingtypes "github.com/lavanet/lava/x/pairing/types"
 )
 
-type RelayFinalizationMetaData struct {
-	MetaData ReplyMetadata
-	Request  pairingtypes.RelayRequest
-	Addr     sdk.AccAddress
+func NewRelayFinalizationMetaDataFromReplyMetadataAndRelayRequest(reply ReplyMetadata, req pairingtypes.RelayRequest, consumerAddr sdk.AccAddress) RelayFinalization {
```

Comment:
this is relayfinalization, not relayfinalizationmetadata
fix method name

Reply:
True

Reply:
Fixed in 0ee11ccb018dd3bfad85eff5846014e458bbdf36